core/modules/replica/sql/replication.sql

```diff
@@ -0,0 +1,793 @@
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 ;
```

Comment:
Great idea! I will implement this.

Reply:
Forgot to mention that this time I would like to avoid code duplication like I did for the ChunkInventory migration (worker side database "qserw_worker"). We probably need to talk how to make this more efficient by putting a common ("standard", "trivial") version of the migration module into some shared module.